lib/components_test.go

```diff
@@ -0,0 +1,39 @@
+package lib
```

Comment:
This comment made me nervous because everyone likes lots of tests lol. I just experimented with this. While the test is in the same package as the code, `go build` does not package *_test.go files into the final binary. The package just is for scope when running `go test`